src/views/List.jsx

```diff
@@ -22,11 +21,13 @@ export function List({ data, listPath }) {
 		navigate(selectedPath);
 	};
 
-	const filteredData = data.filter((item) => {
+	let filteredData = data.filter((item) => {
 		const itemName = item.name.toLowerCase();
 		return itemName.includes(searchTerm.toLowerCase());
 	});
 
+	filteredData = filteredData.sort(comparePurchaseUrgency);
```

Comment:
`.sort()` is mutating the original array, so we don't necessarily need to reassign to the variable again. 

run the code below will be enough, and we could use `const` for `filteredData`
```
filteredData.sort(comparePurchaseUrgency);
```